src/preprocessing/passes/unconstrained_simplifier.cpp

```diff
@@ -425,97 +531,126 @@ void UnconstrainedSimplifier::processUnconstrained()
         {
           bool found = false;
           bool done = false;
-          for(TNode::iterator child_it = parent.begin(); child_it != parent.end(); ++child_it) {
-            if ((*child_it) == current) {
-              if (found) {
+          for (TNode::iterator child_it = parent.begin();
+               child_it != parent.end();
+               ++child_it)
+          {
+            if ((*child_it) == current)
+            {
+              if (found)
+              {
                 done = true;
                 break;
               }
               found = true;
               continue;
             }
-            else if (d_unconstrained.find(*child_it) != d_unconstrained.end()) {
+            else if (d_unconstrained.find(*child_it) != d_unconstrained.end())
+            {
               continue;
             }
-            else {
+            else
```

Comment:
Done. Note: we need to be a bit careful here because we removed the `d_unconstrained.find(*child_it) != d_unconstrained.end()` case, so this has to become `else if (d_unconstrained.find(*child_it) == d_unconstrained.end())`.